### DISCUSSION TITLE
listings_w should NOT enable active listing changes

### DISCUSSION THREAD

**Discussion by KevinMichaelSchindler** [etsy/open-api#1132] [Ideas]
Hello Etsy Team :),

one of my users was concerned about giving my app access to their shop. While the app only creates draft listings I need to ask for the full listings_w permissions which also enables switching the listing state to active via the updateListing endpoint, which incurs costs and exposes the listing to the public. 

It would be very helpful to have a differentiation between the scope of creating a draft and creating/updating an active listing, which is much more sensitive.

Is this possible and do you think it would be a good change?

Warm regards,

Kevin

**Comment by Gareth064** [comment]
Is the concern that they don't want your app to enable listings? Is this a feature of your app? Or does it only create draft listings?

I can't imagine Etsy will do anything to amend the "scope" of the scopes now since the API has been generally available for the year, but you can ask them at developer@etsy.com

**Reply by KevinMichaelSchindler** [reply]
Hey Gareth,

sorry for the late reply! They have security concern when they allow the app to publish listings in their shop on their behalf. My app only creates drafts so I don't need the permission to publish things. However, there is not a  more granular permission set, since the listings_w always enables both. When the user is redirected to grant the permissions, they are asked to grant my app the permission to create AND publish listings which is for them concerning in case of a security breach. I will try to reach out to the developers, maybe I am not the only one :)!

Thanks again for the reply!

Cheers!

Kevin
